src/jolokia/components/JolokiaAuthentification.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC, useState } from 'react';
+import { OpenAPI as OpenAPIConfig } from '../../openapi/jolokia/requests/core/OpenAPI';
+import {
+  useGetApiServerBaseUrl,
+  useJolokiaLogin,
+} from '../../jolokia/customHooks';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { AuthContext } from '../context';
+import { BrokerCR } from '../..//k8s/types';
```

Comment:
Hello @lavocatt , just a small minor update could you please remove the extra `/`  in the import of BrokerCR.